### ISSUE TITLE
[UX] Schedule Design Intent Review

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#49070]
## Tasks
- Update collab cycle ticket with what we have already done
- Schedule a Design Intent review with the Platform
    - See if we can skip the mid-point and IA steps

**Comment by loripusey** [comment]
@ytsaoca @benbrasso-agile6 have we scheduled this yet?

**Comment by benbrasso-agile6** [comment]
Not yet. Working out some decisions for the wireframes. But, I'll get on it.

**Comment by benbrasso-agile6** [comment]
[Created collab cycle ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/49438)

**Comment by benbrasso-agile6** [comment]
Design intent meeting scheduled for 21st. Will work on filling out the ticket.

**Comment by benbrasso-agile6** [comment]
https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/48859

Moving to validate. Just need to update the prorotype link once Zach is finished up.

**Comment by benbrasso-agile6** [comment]
[Experimental design ticket requests were made in Slack](https://dsva.slack.com/archives/C01DBGX4P45/p1668461694875419)

**Comment by benbrasso-agile6** [comment]
Added links to wires

https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/48859
